miio/integrations/vacuum/dreame/tests/test_dreamef9vacuum_miot.py

```diff
@@ -0,0 +1,97 @@
+from unittest import TestCase
```

Comment:
Ah, forgot to add. As the implementation is the same, you don't need a separate test file but you can test only the extra features as you see fit.

Reply:
Please remove this file and just test the extra features inside the already existing test file. I know that the tests are not easy to write and that the already existing tests are not up to par, but if you could just include a test for the `status()` to verify that it's working correctly for both implementations, that would be of great help!